[2.6] Small updates to several guides (#2161)
astronomer/docs: learn/templating.md

)
```

> :::info
> 
> You can find a list of pre-injected functions in the [Airflow documentation](https://airflow.apache.org/docs/apache-airflow/stable/templates-ref.html#macros). You can also load information in JSON format using `"{{ macros.json.loads(...) }}"` and information in YAML format using `"{{ macros.yaml.safe_load(...) }}"`.
> 
> :::
> 

Reviewer: ```suggestion
Airflow includes some pre-injected functions out of the box for you to use in your templates. See [Airflow documentation](https://airflow.apache.org/docs/apache-airflow/stable/templates-ref.html#macros) for a list of available functions. You can also load information in JSON format using `"{{ macros.json.loads(...) }}"` and information in YAML format using `"{{ macros.yaml.safe_load(...) }}"`.
```

> You can find a list of pre-injected functions in the [Airflow documentation](https://airflow.apache.org/docs/apache-airflow/stable/templates-ref.html#macros). Additionally to the listed functions you can load information in JSON format using `"{{ macros.json.loads(...) }}"` and information in YAML format using `"{{ macros.yaml.safe_load(...) }}"`.

Reviewer 2: ```suggestion
You can find a list of pre-injected functions in the [Airflow documentation](https://airflow.apache.org/docs/apache-airflow/stable/templates-ref.html#macros). You can also load information in JSON format using `"{{ macros.json.loads(...) }}"` and information in YAML format using `"{{ macros.yaml.safe_load(...) }}"`.
```
